UnityGLTF/Assets/UnityGLTF/Scripts/Extensions/GLTFSceneExporter.Animation.cs

```diff
@@ -0,0 +1,335 @@
+﻿using System;
+using System.Collections.Generic;
+using GLTF.Schema;
+using UnityEngine;
+using UnityEditor.Animations;
+using UnityEditor;
```

Comment:
That's right

Reply:
One way to do this is to add an editor script that parses through the animation curves in editor mode, stores them somewhere as an exposed variable, then in runtime mode get that information. This is redundant, but would at least allow for "runtime" export.. thoughts?

Reply:
I don't have a good solution for this. Maybe we need to ask Unity on this one.

Reply:
How can we do that?

Reply:
We can try the forums or I can reach out.

Reply:
I checked some recent threads and most people seem to repeat the fact that this not currently feasible. It might be better to reach out to someone from Unity directly, if possible.